runtime/amplitude/src/xcm_config.rs

```diff
@@ -80,30 +65,32 @@ pub struct CurrencyIdConvert;
 
 impl Convert<CurrencyId, Option<MultiLocation>> for CurrencyIdConvert {
 	fn convert(id: CurrencyId) -> Option<MultiLocation> {
-		match id {
-			CurrencyId::XCM(index) => match index {
-				xcm_assets::RELAY_KSM => Some(MultiLocation::parent()),
-				xcm_assets::ASSETHUB_USDT => Some(asset_hub::USDT_location()),
-				_ => None,
-			},
-			CurrencyId::Native => Some(native_location_external_pov()),
-			_ => None,
-		}
+		AssetRegistry::metadata(&id)
+			.filter(|m| m.location.is_some())
```

Comment:
If there exist a location, we assume the asset is transferable. More specifically, that we allow it for paying fees.